frontend/packages/shared/src/hooks/useStudioUrlParams.ts

```diff
@@ -0,0 +1,11 @@
+import { useParams } from 'react-router-dom';
+import type { StudioUrlParams } from 'app-shared/types/StudioUrlParams';
```

Comment:
Kunne vurdert om det er nødvendig å legge denne globalt i types, eller om den er så pass tett knyttet til denne hooken at den naturlig kunne vært en intern type. :)

Reply:
Godt poeng, kan heller ta den ut hvis vi får bruk for den eksternt noen gang. Fikser!